src/components/ModalPortal.js

```diff
@@ -309,6 +309,8 @@ export default class ModalPortal extends Component {
     const { className, overlayClassName, defaultStyles } = this.props;
     const contentStyles = className ? {} : defaultStyles.content;
     const overlayStyles = overlayClassName ? {} : defaultStyles.overlay;
+    const ariaModal = (navigator.userAgent.match(/iPhone/i)) || (navigator.userAgent.match(/iPad/i))
```

Comment:
Please verify for "Mac OS" too, issue appear though all apple devices that use VoiceOver screen reader